vyper/ast/natspec.py

```diff
@@ -8,7 +8,7 @@
 from vyper.parser.global_context import GlobalContext
 from vyper.signatures import sig_utils
 
-SINGLE_FIELDS = ("title", "author", "notice", "dev")
+SINGLE_FIELDS = ("title", "author", "license", "notice", "dev")
```

Comment:
Also need to add `license` to the `invalid_fields` arg when calling `_parse_docstring` on function-level calls.  And it would be good to have a test case for this behavior.

Reply:
Aye